packages/mobile/src/qrcode/QRScanner.tsx

```diff
@@ -47,6 +50,36 @@ const SeeThroughOverlay = () => {
 export default function QRScanner({ onBarCodeDetected }: QRScannerProps) {
```

Comment:
Opening the QR scanner on a physical device results in a crash for me.
![Screenshot_20210504-120542_Alfajores (dev)](https://user-images.githubusercontent.com/15326838/117035836-c8636980-acd2-11eb-92f5-eec7c71a0d08.jpg)